### PR TITLE
Functional tests - (Refacto) Delete 'uppercaseFirstCharacter()' method

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x
| Description?      | Delete no needed method from all files
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| How to test?      | no need
| Possible impacts? | no

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/25201)

### CODE REVIEW

File: tests/UI/campaigns/functional/BO/13_shopParameters/02_orderSettings/01_orderSettings/general/02_enableGuestCheckout.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ describe('Enable guest checkout', async () => {
     {args: {action: 'enable', exist: true, pwdRequired: false}},
   ];
 
-  tests.forEach((test) => {
+  tests.forEach((test, index) => {
     it(`should ${test.args.action} guest checkout`, async function () {
       await testContext.addContextItem(this, 'testIdentifier', `${test.args.action}GuestCheckout`, baseContext);
```
Comment (boubkerbribri) [inline tests/UI/campaigns/functional/BO/13_shopParameters/02_orderSettings/01_orderSettings/general/02_enableGuestCheckout.js]:
```suggestion
  tests.forEach((test, index) => {
    it(`should ${test.args.action} guest checkout`, async function () {
      await testContext.addContextItem(this, 'testIdentifier', `guestCheckout${index}`, baseContext);
```

File: tests/UI/campaigns/functional/BO/13_shopParameters/02_orderSettings/01_orderSettings/general/03_disableReorderingOption.js [hunk 2]
```diff
@@ -61,7 +61,7 @@ describe('Enable reordering option', async () => {
     {args: {action: 'disable', status: false, reorderOption: true}},
   ];
 
-  tests.forEach((test) => {
+  tests.forEach((test, index) => {
     it(`should ${test.args.action} reordering option`, async function () {
       await testContext.addContextItem(this, 'testIdentifier', `${test.args.action}GuestCheckout`, baseContext);
```
Comment (boubkerbribri) [inline tests/UI/campaigns/functional/BO/13_shopParameters/02_orderSettings/01_orderSettings/general/03_disableReorderingOption.js]:
```suggestion
  tests.forEach((test, index) => {
    it(`should ${test.args.action} reordering option`, async function () {
      await testContext.addContextItem(this, 'testIdentifier', `guestCheckout${index}`, baseContext);
```

File: tests/UI/campaigns/functional/BO/13_shopParameters/03_productSettings/01_productsGeneral/05_forceUpdateFriendlyURL.js [hunk 3]
```diff
@@ -144,12 +133,7 @@ describe('Enable/Disable force update friendly URL', async () => {
     });
 
     it('should update the product name and check the friendly URL', async function () {
-      await testContext.addContextItem(
-        this,
-        'testIdentifier',
-        `UpdateProductAndCheckFriendlyURL${addProductPage.uppercaseFirstCharacter(test.args.action)}`,
-        baseContext,
-      );
+      await testContext.addContextItem(this, 'testIdentifier', `UpdateProductAndCheckFriendlyURL${index}`, baseContext);
```
Comment (boubkerbribri) [inline tests/UI/campaigns/functional/BO/13_shopParameters/03_productSettings/01_productsGeneral/05_forceUpdateFriendlyURL.js:136]:
```suggestion
      await testContext.addContextItem(this, 'testIdentifier', `updateProductAndCheckFriendlyURL${index}`, baseContext);
```

File: tests/UI/campaigns/functional/BO/13_shopParameters/04_customerSettings/01_customers/06_enablePartnerOffers.js [hunk 4]
```diff
@@ -57,7 +57,7 @@ describe('Enable partner offer', async () => {
     {args: {action: 'enable', enable: true}},
   ];
 
-  tests.forEach((test) => {
+  tests.forEach((test, index) => {
     it(`should ${test.args.action} partner offer`, async function () {
       await testContext.addContextItem(this, 'testIdentifier', `${test.args.action}PartnerOffer`, baseContext);
```
Comment (boubkerbribri) [inline tests/UI/campaigns/functional/BO/13_shopParameters/04_customerSettings/01_customers/06_enablePartnerOffers.js]:
```suggestion
  tests.forEach((test, index) => {
    it(`should ${test.args.action} partner offer`, async function () {
      await testContext.addContextItem(this, 'testIdentifier', `partnerOffer${index}`, baseContext);
```

Comment (nesrineabdmouleh) [reply tests/UI/campaigns/functional/BO/13_shopParameters/04_customerSettings/01_customers/06_enablePartnerOffers.js]: Done thx :+1:

**Review by boubkerbribri — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Comment by Progi1984** [comment]
## WAIT FOR 1780 RELEASE

**Comment by matks** [comment]
Re-running the CI

**Comment by matks** [comment]
CI failing because of https://github.com/PrestaShop/PrestaShop/issues/25317 . PR not related to https://github.com/PrestaShop/PrestaShop/issues/25317

QA team has defined the scope of changes that need to be embedded inside 1.7.8.0 RC1 build 2. This PR is inside the list.

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thank you @nesrineabdmouleh

**Pull request merged**